src/modules/variable/constinit.rs

```diff
@@ -0,0 +1,73 @@
+use heraclitus_compiler::prelude::*;
+use crate::docs::module::DocumentationModule;
+use crate::modules::types::{Typed, Type};
+use crate::modules::expression::expr::Expr;
+use crate::translate::module::TranslateModule;
+use crate::utils::metadata::{ParserMetadata, TranslateMetadata};
+use super::{variable_name_extensions, handle_identifier_name};
+
+#[derive(Debug, Clone)]
+pub struct ConstInit {
+    name: String,
+    expr: Box<Expr>,
+    global_id: Option<usize>,
+    is_fun_ctx: bool
+}
+
+impl ConstInit {
+    fn handle_add_const(&mut self, meta: &mut ParserMetadata, name: &str, kind: Type, tok: Option<Token>) -> SyntaxResult {
+        handle_identifier_name(meta, name, tok)?;
+        self.global_id = meta.add_var(name, kind, true);
+        Ok(())
+    }
+}
+
+impl SyntaxModule<ParserMetadata> for ConstInit {
+    syntax_name!("Constant Initialization");
```

Comment:
The equivalent line in `init.rs` is:
```
syntax_name!("Variable Initialize");
```
Consider changing one or the other to match.